### ISSUE TITLE
Suggestion - taller end

### ISSUE THREAD

**Issue by beltxd** [jbredwards/Nether-API#10]
Much like having a taller nether roof, could there be a taller end? For example, allow generation of end islands at variable Y levels, in a way, resembling nullscapes mod. If this feature were to be added, would there be any conflicts with mods using nether api?

![image](https://github.com/user-attachments/assets/35534a06-1082-4fc1-a9fc-0bec00875f8a)

![image](https://github.com/user-attachments/assets/13099fb2-42cf-4dd5-95d5-3105e22a730d)

Doesn't have to be exactly like this, but a more general idea of an elevated island generation

**Comment by Foreck1** [comment]
I agree that anything that could add some degree of varibility to the verticality of the end would be huge for making the dimension orders of magnitude more interesting to traverse. Right now, there's a single universal floor across the entire dimension, even though they are islands on the void; it's a huge wast of potential.
Since NetherAPI operates at the core of world generation, offering some degree of Chunkprodivder customization would be **massive**.
